src/main/wrapper/ExecutionService.ts

```diff
@@ -153,6 +153,7 @@ export class ExecutionService {
                 logger.info("Exit code received from AST-CLI: " + code);
                 if(code==1){
                     stderr = stdout
+                    reject(stderr)
```

Comment:
Do we have tests to test this behaviour?

Reply:
No, we do not have any tests in VSCode except UI tests that using mocks.
I think Miryam and Sara will work next sprint on a design for VSCode tests platform